docs/tutorials/developer-journey/level-2/2.1-storage-persistence.md

```diff
@@ -0,0 +1,189 @@
+# 2.1: Canister upgrades, storage, and persistence 
+
+## Overview
+
+The Internet Computer handles long-term data storage using a feature known as **stable memory**. **Stable memory** is a unique feature of the Internet Computer that defines a data store separate from the canister's regular Wasm memory data store, which is known as **heap storage**. A canister's heap storage is not long-term storage and does not persist across canister upgrades; canister data and state stored in heap storage is removed when a canister is stopped, reinstalled, or upgraded. Stable memory is important since it persists across canister upgrades, has a much larger storage capacity than heap storage, and is very beneficial for vertically scaling a dapp. 
```

Comment:
```suggestion
The Internet Computer handles long-term data storage using a feature known as **stable memory**. **Stable memory** is a unique feature of the Internet Computer that defines a data store separate from the canister's regular Wasm memory data store, which is known as **heap storage**. A canister's heap storage is not long-term storage and does not persist across canister upgrades; canister data and state stored in heap storage is removed when a canister is upgraded or reinstalled. Stable memory is important since it persists across canister upgrades, has a much larger storage capacity than heap storage, and is very beneficial for vertically scaling a dapp. 
```